src/prob/test.jl

```diff
@@ -91,6 +91,65 @@ function _post_uc_opf(pm::GenericPowerModel)
     end
 end
 
+"opf with unit commitment, tests constraint_current_limit"
+function _run_uc_strg_opf(file, model_constructor, solver; kwargs...)
```

Comment:
I am thinkinking that this model variant does not bring us too much utility in terms of test coverage.  I propose to remove it.  It seems like all of the functionalities of this model are covered by the other models, or did I miss something?

The general goal of these tests is to exercise all of the code's features with a small collection of problem variants.

Reply:
@ccoffrin 
This test is the regular uc_opf problem, with added storage.  Because storage is added, variable_storage_on_off(), variable_storage_indicator(), and constraint_storage_on_off() are tested.

Once storage is added to the general problems, this test would no longer be necessary.  

Should this test still be included?

Reply:
the strg was merged into uc_opf and uc_mc_opf problems.